### ISSUE TITLE
does the bsp have power vr gpu driver?

### ISSUE THREAD

**Issue by Nigelwz** [renesas-rcar/meta-renesas#37]
Hi can I know the bsp have power vr gpu driver?

**Comment by DuyDang007** [comment]
Hi @Nigelwz,
Which SoC you are mentioning about?

**Comment by Nigelwz** [comment]
About rcar m3 and h3

**Comment by Nigelwz** [comment]
Thanks

**Comment by DuyDang007** [comment]
Yes they have. You can check detail SoC specs in this site: https://www.renesas.com/sg/en/products/automotive-products/automotive-system-chips-socs

**Comment by Nigelwz** [comment]
Does the rcar bsp have gpu source code?
 e.g: driver/gpu/drm/xxxx.c?

**Comment by DuyDang007** [comment]
For such information (driver, source code), please contact our sales https://www.renesas.com/sg/en/contact-us
